http/headers/origin.json

```diff
@@ -23,17 +23,6 @@
               {
                 "version_added": "70"
               },
-              {
-                "version_added": "59",
-                "flags": [
-                  {
-                    "name": "network.http.sendOriginHeader",
-                    "type": "preference",
-                    "value_to_set": "true"
-                  }
-                ],
-                "notes": "Sent with non-CORS requests since Fx59, unless they are <code>GET</code> or <code>HEAD</code>."
-              },
               {
                 "version_added": true,
```

Comment:
Here's an "ambiguous gap" that the guideline talks about.

Reply:
I mean, technically, we're not _only_ leaving behind a `version_added: true`; we _do_ have `version_added: 70` up above, so this _is_ technically within the guidelines?  😝 

(I definitely would want to try and find a version number for this, but I'm not yet certain how to find version numbers for HTTP headers and I wouldn't want our flag removal to be blocked on it...)

Reply:
OK, let's leave this a mystery for the future :)